server/field/lifepool/mob/mob.go

```diff
@@ -63,14 +66,19 @@ type Data struct {
 	link                   int32
 	flySpeed               int32
 	noRegen                int32
-	skills                 map[byte]byte
+	Skills                 map[byte]byte
```

Comment:
Should this be private?

Reply:
This variable needed to be global for server/field/lifepool/pool.go #288 to work. I can make a global function instead that returns the Skills map.